bin/install-theos

```diff
@@ -361,9 +361,9 @@ linux() {
 
 	# Check for WSL
 	update "Checking for WSL..."
-	if [[ $(uname -r | sed -n 's/.*\( *Microsoft *\).*/\L\1/ip') == microsoft ]]; then
-		VERSION=$(uname -r | sed 's/.*\([[:digit:]]\)[[:space:]]*/\1/')
-		if [[ $VERSION -eq 1 ]]; then
+	rel="$(uname -r)"
+	if [[ ${rel,,} =~ microsoft ]]; then
```

Comment:
What does `${rel,,}` do? This doesn't work on macOS bash (this code is in the `linux` block, so if there's an incompatibility, I think that's ok).

```
$ bash --version
GNU bash, version 3.2.57(1)-release (arm64-apple-darwin23)
Copyright (C) 2007 Free Software Foundation, Inc.
$ rel="$(uname -r)"
$ if [[ ${rel,,} =~ microsoft ]]; then
    echo true
  fi
bash: ${rel,,}: bad substitution
```

Reply:
Sets the string to lowercase. And, yup, is a GNU bash feature I believe

Reply:
Okay, thanks. Sounds good to me.